python/lsst/ts/rubintv/__init__.py

```diff
@@ -0,0 +1,32 @@
+# This file is part of ts_mtdome.
```

Comment:
Please refer to `rubintv` instead of `ts_mtdome` here and in all other Python file headers.

Reply:
jeje too much copy/paste, will do the change :+1:.